apps/src/lib/ui/SchoolInfoConfirmationDialog.jsx

```diff
@@ -4,11 +4,24 @@ import Dialog, {Body} from '@cdo/apps/templates/Dialog';
 import Button from '../../templates/Button';
 import SchoolInfoInterstitial from './SchoolInfoInterstitial';
 import i18n from '@cdo/locale';
+import color from '@cdo/apps/util/color';
 
 export const styles = {
   button: {
     marginTop: 30,
     marginLeft: 290
+  },
+  updateButton: {
```

Comment:
My CSS knowledge is subpar but should this be `update-button`?

Reply:
Good question.  In React, it is camel case.